wiki/Tournaments/LMS/Shift/1/en.md

```diff
@@ -28,10 +28,8 @@ The **Taiko LMS: Shift Cup** is a 3 on 3, 1v1 knock-out style osu!taiko tourname
 | Placing | Prize(s) |
 | --- | :-- |
 | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Special profile badge |
```

Comment:
Looking that the 2nd and 3rd places will get nothing, i think it would be better to write a sentence or two saying that the winner of the tournament will get the a profile badge (you can also put that badge here), better than doing a table and filling the prizes with "none"